telemetry/packages/public-app/components/accelerationChart.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { Card, LineChart, Title } from '@tremor/react';
+import { useQuery } from 'react-query';
+import format from 'date-fns/format';
+
+const valueFormatter = (number: any) =>
+  `$ ${new Intl.NumberFormat('us').format(number).toString()}`;
+
+export const AccelerationChart = () => {
+  const { data, isLoading, isError } = useQuery(
+    'displacement    ',
+    async () =>
+      (await fetch(
+        `${process.env.NEXT_PUBLIC_TELEMETRY_SERVER}/pods/pod_1/public-data/displacement?start=0`,
+      ).then(res => res.json())) as {
+        id: 'displacement';
+        timestamp: string;
+        value: number;
+      }[],
+    {
+      refetchInterval: 1000,
+    },
+  );
+
+  const displacementData = data
+    ? data.map(d => {
+        const time = new Date(d.timestamp);
+        return {
+          time: format(time, 'HH:mm:ss'),
+          displacement: d.value,
+        };
+      })
+    : [];
+
+  //if (isLoading) return <div>Currently loading acceleration...</div>;
```

Comment:
We can create a custom loading and error state if we slightly change how we load the data initially in order to use Next.js 's Loading UI and Streaming (https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming). We can look at this together :) Basically we'll do an initial fetch and while that happens we can define a loading component. If it errors (or we throw an error when React Query returns an error) then we can display a custom error component instead.